docs/network/index.html

```diff
@@ -854,25 +854,62 @@ <h2 id="methods">Methods</h2>
                 <td colspan="2">Methods to get information on nodes and edges.</td>
             </tr>
             <tr class="collapsible toggle" onclick="toggleTable('methodTable','getPositions', this);">
-                <td colspan="2"><span parent="getPositions" class="right-caret" id="method_getPositions"></span> getPositions(<code><i>[Array of
-                    nodeIds]</i></code>)
+                <td colspan="2"><span parent="getPositions" class="right-caret" id="method_getPositions"></span>
+                    getPositions(<code><i>[Array of nodeIds]</i></code>, <code><i>[nodeId]</i></code>)
```

Comment:
This looks like two optional parameters, ie:
```typescript
getPositions(nodes: NodeId[], nodeId: NodeId): unknown
```

Reply:
Totally right! Should be fixed with the latest push.